locustempus/settings_docker.py

```diff
@@ -1,12 +1,21 @@
 # flake8: noqa
 from locustempus.settings_shared import *
-from ccnmtlsettings.docker import common
 import os
 
-locals().update(
-    common(
-        project=project,
-        base=base,
-        STATIC_ROOT=STATIC_ROOT,
-        INSTALLED_APPS=INSTALLED_APPS,
-    ))
+# docker-compose db container
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql',
```

Comment:
Django needs to use the postgis db backend like the rest of the settings files in this project, so this should be changed to:
```
django.contrib.gis.db.backends.postgis
```